cypress/integration/chi-custom-elements/icon.spec.js

```diff
@@ -0,0 +1,58 @@
+/// <reference types="Cypress" />
+
+describe('Icons', function() {
+
+  beforeEach(()=>{
+    cy.fixture('chidata.json').as('chidata');
+  });
+
+  it('Base icon should be included inside the custom element', function() {
+    cy.visit('http://localhost:8000/tests/custom-elements/icons.html');
```

Comment:
I think we should have this URL on cypress environment file in order to run this code on different environments.